exp/api/v1alpha3/azuremanagedcontrolplane_types.go

```diff
@@ -86,6 +88,14 @@ type AzureManagedControlPlaneStatus struct {
 	// In the AzureManagedControlPlane implementation, these are identical.
 	// +optional
 	Initialized bool `json:"initialized,omitempty"`
+
+	// VirtualNetwork is the name of the AKS-generated VNET. It's not predictable unless we pre-create it.
+	// Consider pre-creating?
+	VirtualNetwork string `json:"virtualNetwork,omitempty"`
```

Comment:
Slight preference for `VirtualNetworkName` or `VirtualNetwork.Name` over `VirtualNetwork string`